internal/store/utils.go

```diff
@@ -123,10 +123,10 @@ func mapToPrometheusLabels(labels map[string]string, prefix string) ([]string, [
 }
 
 func labelName(prefix, labelName string) string {
-	return prefix + "_" + lintLabelName(sanitizeLabelName(labelName))
+	return prefix + "_" + lintLabelName(SanitizeLabelName(labelName))
 }
 
-func sanitizeLabelName(s string) string {
+func SanitizeLabelName(s string) string {
```

Comment:
`exported: exported function SanitizeLabelName should have comment or be unexported (revive)`